server/storage/mvcc/kvstore_compaction.go

```diff
@@ -20,7 +20,7 @@ import (
 	"time"
 
 	"go.uber.org/zap"
-
+	humanize "github.com/dustin/go-humanize"
```

Comment:
also put this item right above `"go.uber.org/zap"`
```suggestion
	humanize "github.com/dustin/go-humanize"

```